docs/topics/create-first-yaml-rule.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="create-first-yaml-rule_{context}"]
+= Creating your first YAML rule
+
+This section guides you through the process of creating and testing your first {ProductShortName} YAML-based rule. This assumes that you have already installed {ProductShortName}. See link:{ProductDocUserGuideURL}/index#installing_and_running_the_cli[Installing and running the CLI] in the _{UserCLIBookName}_ for installation instructions.
+
```

Comment:
Assumes you have "MTA CLI" (not just "MTA") installed, right? Really is a prerequisite, but I won't insist you label it that way now.